xonsh/prompt/cwd.py

```diff
@@ -28,7 +28,13 @@ def _replace_home(x):
 
 
 def _replace_home_cwd():
-    return _replace_home(builtins.__xonsh_env__['PWD'])
+    env = builtins.__xonsh_env__
+    if 'RESOLVE_SYMLINKS' in env and not env['RESOLVE_SYMLINKS'] \
```

Comment:
I don't think you have to check that `RESOLVE_SYMLINKS` and `LITERAL_PWD` are in the environment.